### ISSUE TITLE
[warlock] pet ability and combat 

### ISSUE THREAD

**Issue by mipanka** [gamefreedomgit/Maelstrom#5660]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**
succubus can use https://wowpedia.fandom.com/wiki/Lesser_Invisibility when warlock in combat

**How to reproduce:**
input in combat and use spell

**How it should work:**
succubus should spam invisibility when warlock in combat

**Database links:**
https://wowpedia.fandom.com/wiki/Succubus_(warlock_minion)
https://wowpedia.fandom.com/wiki/Lesser_Invisibility

https://youtu.be/mkWvdohVk9Y?t=131 any time when warlock in combat this ability cannot be used

**Comment by mipanka** [comment]
Invisible cannot be used when pet in combat.
https://wowwiki-archive.fandom.com/wiki/Seduction to use and break cast this pet spell, when warlock in combat. For break invisible, and succubus will spam invisible spell after every break cast. Bug with pet combat I think.

**Comment by UltraNix** [comment]
Seduction does not put pet in combat nor prelong pvp combat.
Please, record that bug so I'm sure what are we talking about.

**Comment by mipanka** [comment]
yes, but as I see in video https://youtu.be/5WWRot09tN8?t=87 when warlock in combat he will input pet too https://youtu.be/5WWRot09tN8?t=468 here pet casted seduction and got block, but not when warlock got combat, mb official server delay idk

**Comment by UltraNix** [comment]
So as I asked you - record a video from our server.

**Comment by mipanka** [comment]
https://www.youtube.com/watch?v=84bkYZc9Ggs&ab_channel=%D0%94%D0%BC%D0%B8%D1%82%D1%80%D0%B8%D0%B9 
in game now

**Comment by UltraNix** [comment]
@mipanka Combat on warlock is not the same as combat on pet.
Putting combat with warlock does not put you in combat with pet.

**Comment by UltraNix** [comment]
As I can see on the video, damage breakes invisibility, but do not put pet in combat.
Combat with warlock does not transfer automatically to pet. You can be in combat with warlock, but not with the pet.
